IFC4x3/PropertyConstants/s/SLIPTURNOUTANDSCISSORSCROSSING _39enOhW$T4iBh9ig4eHZFN.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SLIPTURNOUTANDSCISSORSCROSSING__39enOhWT4iBh9ig4eHZFN" Name="SLIPTURNOUTANDSCISSORSCROSSING " UniqueId="c9a3162b-83f7-44b0-bac9-b2a1284633d7" />
```

Comment:
Difficult to read.

Reply:
I have added underscore. Commit : 426664c10f4b7a9f9b1eff4dddaa4ee62a65d0e3